packages/agents-hosting/src/auth/jwt-middleware.ts

```diff
@@ -43,7 +43,7 @@ const verifyToken = async (raw: string, config: AuthConfiguration): Promise<JwtP
 
   return await new Promise((resolve, reject) => {
     const verifyOptions: jwt.VerifyOptions = {
-      issuer: config.issuers,
+      issuer: config.issuers as [string, ...string[]],
```

Comment:
@kylerohnmsft after updating dependencies, I got the same error. And fixed with that casting. fyi